src/editor/validator.ts

```diff
@@ -24,7 +23,6 @@ export class Validator {
         const context = providedContext ? providedContext : this.module.focus.getContext();
 
         return (
-            context.token instanceof EmptyExpr ||
```

Comment:
This is part of the discussion about separating the Validator from the Type-Checker.